environment/src/test/java/com/sequenceiq/environment/environment/validation/network/AzureEnvironmentNetworkValidatorTest.java

```diff
@@ -295,7 +295,7 @@ void testValidateDuringRequestWhenNoNetworkCidrAndNetworkId() {
 
     @Test
     void testValidateWhenNoNetworkCidrAndNoNetworkId() {
-        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(new AzureParams(), null, null, null, null, 1);
+        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(AzureParams.builder().build(), null, null, null, null, 1);
```

Comment:
builder -> Builder ?

Reply:
No. `builder()` is a static method of the POJO that returns a builder object.